.github/workflows/jsdom-ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         node-version:
           - 18
           - 20
-          - latest
+          - 22
```

Comment:
It should be
- 18
- 20
- 22
- latest

Reply:
Added that now